### DISCUSSION TITLE
Facing confusion on OSSEC and Wazuh

### DISCUSSION THREAD

**Discussion by rohan0017** [Security-Onion-Solutions/securityonion#11507] [General] [answered]
Q1] On security onion Alerts page I am seeing alerts from OSSEC module, I know that wazuh is a fork of OSSEC. But I have never seen alerts from wazuh module(as it is also a part of Security onion).

Q2]  Why both Wazuh and OSSEC are there in Security onion, is it because Wazuh has more good features than OSSEC.

Can we change the event.module name from ossec to wazuh?

Can anyone help me understand  whether there is any type of connection between both, or they are individual components.

**Comment by dougburks** [comment, marked as answer]
> Q1] On security onion Alerts page I am seeing alerts from OSSEC module, I know that wazuh is a fork of OSSEC. But I have never seen alerts from wazuh module(as it is also a part of Security onion).

Wazuh is a fork of OSSEC but it still uses the OSSEC name in many places.

> Q2] Why both Wazuh and OSSEC are there in Security onion, is it because Wazuh has more good features than OSSEC.

It's not that we include both Wazuh and OSSEC, we only include Wazuh but it still uses the OSSEC name in many places.

Please note that we no longer include Wazuh (or OSSEC) as of Security Onion 2.4. We've announced the End Of Life date for Security Onion 2.3 (which includes Wazuh/OSSEC support), so you should start making plans to migrate to 2.4 now:
https://blog.securityonion.net/2023/10/6-month-eol-notice-for-security-onion-23.html

**Reply by rohan0017** [reply]
Is it possible to change the event.module name from ossec to wazuh(just curious)? If not why not.

**Reply by rohan0017** [reply]
Yes I am planning to migrate to Security Onion 2.4. Can I know what happens to Security Onion 2.3 after it reaches EOL.

**Reply by dougburks** [reply]
> Is it possible to change the event.module name from ossec to wazuh(just curious)? If not why not.

You might be able to make this work, but it's not something that we recommend or support.

> Yes I am planning to migrate to Security Onion 2.4. Can I know what happens to Security Onion 2.3 after it reaches EOL.

Once Security Onion 2.3 reaches EOL, it will no longer be maintained or supported.

**Reply by rohan0017** [reply]
Thank you very much for the answers.
